index.css

```diff
@@ -66,4 +66,25 @@ body {
     color: #ffffff;
     background-color: #10859e;
     border-color: #0e758b;
-}
\ No newline at end of file
+}
+
+#uv-container {
+    position: relative;
+}
+
+#logout {
+    top: 0;
+    right: 0;
+    position: absolute;
+    padding: 10px;
+}
+
+#logout a {
+    font-size: 12px;
+    font-weight: normal;
+    line-height: 1.42857143;
```

Comment:
that's 20/14 right?

Reply:
Yeah, I copied it from one of the existing links they've styled.